packages/core/test/integration/model.test.js

```diff
@@ -439,15 +439,14 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         });
       }
 
+      const ModelOptions = ['mariadb', 'mysql'].includes(dialectName) ? { indexes: indices, engine: 'MyISAM' } : { indexes: indices };
```

Comment:
I was wondering about this; why do we set the SUPPORTED_OPTIONS and SUPPORTABLE_OPTIONS in a set instead of using DialectSupports like `supports.createTable.engine`?

Reply:
I was thinking the same thing. We should investigate how we could merge them